Makefile

```diff
@@ -184,25 +285,71 @@ tools/.git:
 $(RUNNER_EXE): tools/.git
 	make -C tools/runner
 
+# CODE FOR PUBLISHING THE IMAGES
+
+# This target first runs "make publish" for each architecture
+# Then it pushes the manifest lists
+publish: ./manifest-tool $(addprefix sub-publish-,$(ALL_ARCHITECTURES))
+	$(MAKE) DOCKER_HOST=$(DOCKER_HOST) DOCKERHUB_USER=$(DOCKERHUB_USER) WEAVE_VERSION=$(WEAVE_VERSION) UPDATE_LATEST=$(UPDATE_LATEST) $(PUSH_ML)
+ifeq ($(PUBLISH_WEAVEDB),true)
+	$(SUDO) DOCKER_HOST=$(DOCKER_HOST) docker push $(DOCKERHUB_USER)/weavedb:latest
+endif
+
+sub-publish-%:
+	$(MAKE) DOCKER_HOST=$(DOCKER_HOST) clean-bin
+	$(MAKE) ARCH=$* DOCKER_HOST=$(DOCKER_HOST) DOCKERHUB_USER=$(DOCKERHUB_USER) WEAVE_VERSION=$(WEAVE_VERSION) UPDATE_LATEST=$(UPDATE_LATEST) $(IMAGES_UPTODATE)
+	$(MAKE) ARCH=$* DOCKER_HOST=$(DOCKER_HOST) DOCKERHUB_USER=$(DOCKERHUB_USER) WEAVE_VERSION=$(WEAVE_VERSION) UPDATE_LATEST=$(UPDATE_LATEST) publish-one-arch
+
+./manifest-tool:
+	wget https://github.com/luxas/manifest-tool/releases/download/v0.3.0/manifest-tool
```

Comment:
Works for me, should I send a PR against build-tools with a submodule to luxas/manifest-tool?